styles/styles.css

```diff
@@ -0,0 +1,127 @@
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
```

Comment:
No need for this 2 :smile:

Reply:
you're right, its the same code as in the equalizer.css, but for some reason if i remove it, the default margin on the left appears again. 
![image](https://user-images.githubusercontent.com/34032882/33488032-1a4a76cc-d68e-11e7-9704-591ef1157a22.png)
i even tried removing my styles.css from the html and leaving only the equalizer, and it is still there
![image](https://user-images.githubusercontent.com/34032882/33487970-e378a164-d68d-11e7-8afb-f60568148380.png)
smh i have no idea what is going on

Reply:
Really weird :thinking: